ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -244,10 +244,37 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      TEST_SCENARIOS: MCO|Cluster_Infrastructure
+      TEST_SCENARIOS: Cluster_Infrastructure
     test:
     - chain: openshift-e2e-test-qe-longduration
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-f28-longduration-mco-critical
```

Comment:
If f28 is Fedora 28, you may want to just name it with fedora so you can update the Fedora version without renaming the job and thus losing all your history in associated tooling. We don't typically pin a RHEL version in job names and we have rolled out new major versions within one release and job name.

Is longduration useful in the job name?

Reply:
Hello! as far as I know "f28" is not related with "fedora 28". I think it is related to the execution frequency, I used the same naming convention used in the original job. Please, @rioliu-rh @kuiwang02 could you confirm that the "f28" part of the name is correct?

`Longduration` makes sense since it is a reference to the script that executes the tests. Instead of using the default test execution script we are using the "longduration" execution script.

Reply:
yes, I can confirm that `f28` is not related to fedora version, it is used to identify the job frequency

Reply:
@dgoodwin not sure above answers are ok for you, can you review again, thx